examples/warehouse/domain/README

```diff
@@ -41,3 +45,10 @@ dns.yaml:
       ns:
         - IP address of NS server authoritative for this domain
         - ...
+
+services.yaml:
+  domain:
+    services:
+      syslog:
+        - A central Syslog server, in the form <fqdn>:<port>
+        - ...
```

Comment:
The `<fqdn>:<port>` format lacks protocol information, and probably requires string parsing when generating tool specific configuration. Separate data items into a dictionary.
```yaml
net:
  service:
    syslog:
      - address: syslog.example.com
        port: 514
        protocol: udp
```